portal_sale_mail_link/i18n/es.po

```diff
@@ -0,0 +1,50 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* portal_sale_mail_link
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-09-29 09:38+0000\n"
+"PO-Revision-Date: 2020-09-29 09:38+0000\n"
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: portal_sale_mail_link
+#: code:addons/portal_sale_mail_link/models/mail_template.py:21
+#, python-format
+msgid "\n"
```

Comment:
It should be the translation, not the original text

Reply:
Thanks @ernestotejeda :)